object_tracker/custom_tracker.py

```diff
@@ -0,0 +1,75 @@
+from collections import OrderedDict
+from pytracking.evaluation import Tracker
+from pytracking.evaluation.multi_object_wrapper import MultiObjectWrapper
+
+
+
+class CustomTracker(Tracker):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    def run_frame(self, frame):
+        """
+        the method receive a frame as input and return
+        [min_x, min_y, max_x, max_y], flag, score(float)
+        args:
+            frame: image
+        """
+        if frame is None:
+            return None
+        out = self.tracker.track(frame)
+        state = [int(s) for s in out["target_bbox"][1]]
+        min_x = state[0]
+        min_y = state[1]
+        max_x = state[2] + state[0]
+        max_y = state[3] + state[1]
+        return min_x, min_y, max_x, max_y, out["flag"][1], out["score"][1]
+
+    def init_tracker(self, frame, bounding_box):
+        """
+        the method init the tracker
+        args:
+            frame: image
+            bounding_box: [x,y,w,h]
+        """
+        params = self.get_parameters()
+        debug_ = getattr(params, "debug", 0)
+        params.debug = debug_
+
+        params.tracker_name = self.name
+        params.param_name = self.parameter_name
+        multiobj_mode = getattr(
+            params,
+            "multiobj_mode",
+            getattr(self.tracker_class, "multiobj_mode", "default"),
+        )
+        if multiobj_mode == 'default':
```

Comment:
i remove the multiobj options and stay only with default,
it require more changes in the code 
i think it would be good if another person will look on this

Reply:
Do we want Y to take a look?

Reply:
yes